internal/cephfs/core/fsjournal.go

```diff
@@ -124,6 +124,11 @@ func CheckVolExists(ctx context.Context,
 			return nil, cerrors.ErrClonePending
 		}
 		if cloneState == cephFSCloneFailed {
+			log.ErrorLog(ctx,
```

Comment:
"clone failed, deleting subvolume clone. vol=%s, subvol=%s subvolgroup=%s". Maybe specifying the names will be more clearer?

Reply:
i can do that but we dont follow any specific format for cephfs logging.

Reply:
This seems more clear to be but yeah it's fine for me either way.